user/views.py

```diff
@@ -0,0 +1,53 @@
+import jwt
+import json
+import bcrypt
+
+from django.shortcuts import render
+from django.views     import View
+from django.db        import transaction
+from django.http      import JsonResponse, HttpResponse
+
+from django.core.validators import MinLengthValidator
+
+from .models          import User, Point
+from decorator        import login_check
+from my_settings      import SECRET_KEY
+
+# 회원가입
+class SignUpView(View):
+    @transaction.atomic
+    def post(self, request):
+        try:
+            data            = json.loads(request.body)
+            password        = data['password'].encode('utf-8')
+            hashed_password = bcrypt.hashpw(password, bcrypt.gensalt()).decode('utf-8')
```

Comment:
```suggestion
            hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```
password 변수가 재사용 될게 아니라면 줄여주는게 더 좋을것 같아요.

Reply:
넵! 줄이겠습니다~

Reply:
앗 아래에서 유효성 검사로 길이를 체크하기 때문에 사용해야할 거 같습니다!

Reply:
넵 그럼 오케이